### ISSUE TITLE
Plugin keyboard: sometimes the keyboard does not appear always when you click edit

### ISSUE THREAD

**Issue by silviabf** [teltek/Galicaster#293]
Steps:
- Go to conf.ini and configure Keyboard = True
- Launch galicaster
- Wait until keyboard disappear of the screen
- Go to media manager and click on Edit in any recording

Problem: When you click on edit, the keyboard does not appear again ( if you edit something before the keyboard disappear the first time, it works correctly)

**Comment by hcaloto** [comment]
Fixed in https://github.com/teltek/Galicaster/commit/1ce4958932919b729fe67db64ff4745ed4728771

**Comment by silviabf** [comment]
The bug is not solved. The keyboard does not disappear, and sometimes you have a segfault of unity
Message in the console:
```
** (onboard:11139): WARNING **: AT-SPI: Error in GetItems, sender=(null), error=Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
/usr/lib/python3/dist-packages/Onboard/OnboardGtk.py:149: Warning: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
  Gtk.main()

** (onboard:11139): WARNING **: AT-SPI: Error in GetItems, sender=org.freedesktop.DBus, error=The name :1.316 was not provided by any .service files

** (onboard:11139): WARNING **: AT-SPI: Error in GetItems, sender=org.freedesktop.DBus, error=The name :1.317 was not provided by any .service files
```
Message in dmesg:
`[mié jun 15 16:54:31 2016] unity-panel-ser[1409]: segfault at 61 ip 0000000000406ea0 sp 00007ffce71ca200 error 4 in unity-panel-service[400000+12000]`

**Comment by dpeite** [comment]
Fixed in #325

**Comment by silviabf** [comment]
It appears always you click "edit" in Galicaster now. I close this issue.

**Comment by Alfro** [comment]
This does not work anymore.
Doing a git bisect shows we broke it with 3e7d6e0b215d8a446277390eddf12a5fdc8c4c27, which fixed #321.

The current issue is an [Onboard bug](https://bugs.launchpad.net/onboard/+bug/1627819), where it does not seem to work very well with fullscreen applications. Before, opening the "Edit metadata" dialog would throw Galicaster out of fullscreen while the dialog was open, which was a bug, but would provoke the keyboard to appear correctly. Since we fixed that, it does no appear instantly, unless you click outside of the dialog box.
